src/panel/direction/DirectionPanel.jsx

```diff
@@ -224,7 +224,7 @@ export default class DirectionPanel extends React.Component {
       this.setState({ activePreviewRoute: null });
     } else {
       Telemetry.add(Telemetry.ITINERARY_CLOSE);
-      window.app.navigateTo('/');
+      window.history.back();
```

Comment:
Doing that unconditionnaly will exit the app if we arrived directly on the direction panel. We should have a mechanism to return to the home instead in such case.
The common pattern for this is to compare `document.referrer`, which can be empty, filled with the current domain name or another one when coming from the another site. But it's not updated for a SPA, so maybe we're screwed :/